operator/pkg/operator.go

```diff
@@ -165,30 +176,41 @@ func (o *Operator) ProcessNewTaskCreatedLog(newTaskCreatedLog *servicemanager.Co
 	}
 }
 
-func (o *Operator) VerifyPlonkProof(proofBytes []byte, pubInputBytes []byte, verificationKeyBytes []byte) bool {
-	proofReader := bytes.NewReader(proofBytes)
-	proof := plonk.NewProof(ecc.BLS12_381)
-	_, err := proof.ReadFrom(proofReader)
+// VerifyPlonkProofBLS12_381 verifies a PLONK proof using BLS12-381 curve.
+func (o *Operator) VerifyPlonkProofBLS12_381(proofBytes []byte, pubInputBytes []byte, verificationKeyBytes []byte) bool {
+	return o.verifyPlonkProof(proofBytes, pubInputBytes, verificationKeyBytes, ecc.BLS12_381)
+}
```

Comment:
It doesn't make sense to make this function public since it is not going to be called from other places.